### PR TITLE
Update woke.txt

### PR DESCRIPTION
**Pull request by Hexxiiii**
added some extra phrases

### CODE REVIEW

File: woke.txt [hunk 1]
```diff
@@ -21,3 +21,11 @@ pronoun(s)
 meow
 nya+
 nonbinary
+mrrp
+>.<
```
Comment (glistermelon) [inline woke.txt]: Replace `.` with `\.` because this is regex

File: woke.txt [hunk 2]
```diff
@@ -21,3 +21,11 @@ pronoun(s)
...
+uwu
+owo
+:3
+they
```
Comment (glistermelon) [inline woke.txt]: remove they/them because the ordinary 3rd person plural isnt woke...

Comment (Hexxiiii) [reply woke.txt]: okay but itd be funny imo...

(ok...)

File: woke.txt (hunk 1 above)
Comment (Hexxiiii) [reply woke.txt]: legit while adding them i thought about doing it but i wasn't sure if i had to do it, gonna fix

**Pull request merged**